interfaces/apparmor/apparmor.go

```diff
@@ -104,3 +107,8 @@ func LoadedProfiles() ([]Profile, error) {
 	}
 	return profiles, nil
 }
+
+// ProfileName returns the name of apparmor profile for a given application.
+func ProfileName(appInfo *snap.AppInfo) string {
+	return interfaces.SecurityTag(appInfo)
```

Comment:
Importing two previously unnecessary packages here just so it can "rename" a function feels like unnecessary boilerplate, in principle.